packages-exp/auth-exp/src/core/persistence/in_memory.test.ts

```diff
@@ -37,7 +37,7 @@ describe('core/persistence/in_memory', () => {
 
   it('should work with user', async () => {
     const key = 'my-super-special-user';
-    const value = testUser('uid');
+    const value = testUser({} as any, 'uid');
```

Comment:
can these by typed? why do we cast to 'as any'?

Reply:
These values get immediately turned into plain objects. I explicitly don't want these to have an auth object. I've changed the type on testUser to avoid the cast to any